test/functional/uvm_memory_test.go

```diff
@@ -0,0 +1,56 @@
+package functional
+
+import (
+	"context"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/Microsoft/hcsshim/internal/uvm"
+	"github.com/Microsoft/hcsshim/osversion"
+	testutilities "github.com/Microsoft/hcsshim/test/functional/utilities"
+)
+
+func TestUVMMemoryUpdateLCOW(t *testing.T) {
+	testutilities.RequiresBuild(t, osversion.RS5)
+
+	ctx, cancel := context.WithTimeout(context.Background(), 40*time.Second)
+	defer cancel()
+
+	opts := uvm.NewDefaultOptionsLCOW(t.Name(), "")
+	opts.MemorySizeInMB = 1024 * 2
+	u := testutilities.CreateLCOWUVMFromOpts(ctx, t, opts)
+	defer u.Close()
+
+	newMemorySize := uint64(opts.MemorySizeInMB/2) * bytesPerMB
+
+	if err := u.UpdateMemory(ctx, newMemorySize); err != nil {
+		t.Fatalf("failed to make call to modify UVM memory size in MB with: %v", err)
+	}
+}
```

Comment:
Why don't we make the `u.GetAssignedMemoryInBytes(ctx)` call to check if the update actually went through like we do for the WCOW test?

Reply:
Good point. Must have forgotten to update this one. Thanks for the catch.